internals/overlord/servstate/handlers.go

```diff
@@ -825,6 +828,37 @@ func (s *serviceData) checkFailed(action plan.ServiceAction) {
 	}
 }
 
+// writeMetrics writes the service's metrics.
+func (d *serviceData) writeMetrics(writer metrics.Writer) error {
+	err := writer.Write(metrics.Metric{
+		Name:       "pebble_service_start_count",
+		Type:       metrics.TypeCounterInt,
+		ValueInt64: d.startCount.Load(),
+		Comment:    "Number of times the service has started",
+		Labels:     []metrics.Label{metrics.NewLabel("service", d.config.Name)},
+	})
+	if err != nil {
+		return err
+	}
+
+	active := 0
+	if stateToStatus(d.state) == StatusActive {
+		active = 1
+	}
+	err = writer.Write(metrics.Metric{
+		Name:       "pebble_service_active",
+		Type:       metrics.TypeGaugeInt,
+		ValueInt64: int64(active),
+		Comment:    "Indicates if the service is currently active (1) or not (0)",
```

Comment:
Let's use the same phrasing here as for the checks one, so "Whether the service is..."